packages/react-components/src/components/SendBoxErrors.tsx

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import React, { useEffect } from 'react';
+import { SendBoxErrorBar } from './SendBoxErrorBar';
+
+/**
+ * @private
+ * @TODO Add a timestamp to the error
+ */
+export type SendBoxError = {
+  message?: string;
+};
+
+/**
+ * @private
+ */
+export interface SendBoxErrorsProps {
+  fileUploadsPendingError?: SendBoxError;
+  onDismissFileUploadsPendingError?: () => void;
+  fileUploadError?: SendBoxError;
+}
+
+/**
+ * @private
+ */
+export const SendBoxErrors = (props: SendBoxErrorsProps): JSX.Element => {
+  const { fileUploadError, fileUploadsPendingError, onDismissFileUploadsPendingError } = props;
+  const [showFileUploadsPendingError, setShowFileUploadsPendingError] = React.useState(false);
+
+  useEffect(() => {
+    fileUploadsPendingError && setShowFileUploadsPendingError(true);
+  }, [fileUploadsPendingError]);
+
+  // If a new `fileUploadError` is available, show it instead of the `fileUploadsPendingError`
+  useEffect(() => {
+    setShowFileUploadsPendingError(false);
+  }, [fileUploadError]);
+
+  if (showFileUploadsPendingError && fileUploadsPendingError) {
+    return (
+      <SendBoxErrorBar
+        message={fileUploadsPendingError.message}
+        dismissAfterMs={10 * 1000}
+        onDismiss={() => {
+          setShowFileUploadsPendingError(false);
+          onDismissFileUploadsPendingError && onDismissFileUploadsPendingError();
```

Comment:
So for all the other file upload error, we don't need a onDismiss for them
I am thinking of if we could make it more generic for any future needs 

Something like onDismissError(errorType: 'fileUploadPending' | 'uploadFailed') => ()

So we don't have to add another onDismissError for other errors in the future, and we can always do conditional logic to determine what to do